test/mlir/conversion/onnx_to_torch/matmul/matmul3.onnx.mlir

```diff
@@ -0,0 +1,10 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<2x3x4x5xf32>) -> tensor<2x3x4x4xf32> attributes {input_names = ["0"], output_names = ["2"]} {
+    %0 = "onnx.Constant"() {onnx_node_name = "Constant_0", value = dense<"0xDEADBEEF"> : tensor<2x3x5x4xf32>} : () -> tensor<2x3x5x4xf32>
+//CHECK: torch.aten.matmul %arg0, %2 :   
```

Comment:
ditto

Reply:
We have added type check in both the test cases. Please check.